content/docs/manifest.json

```diff
@@ -423,16 +423,7 @@
         },
         {
           "title": "FAQ",
-          "routes": [
-            {
-              "title": "Introduction",
-              "path": "/docs/faq/README.md"
-            },
-            {
-              "title": "TLS Terminology",
-              "path": "/docs/faq/terminology.md"
-            }
-          ]
+          "path": "/docs/faq/README.md"
```

Comment:
FAQ is now a single page rather than a directory with an index page.

Reply:
I very much like that.